cypress/e2e/signIn.cy.js

```diff
@@ -1,34 +1,40 @@
+/* eslint-disable */
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-import SignInPageObject from '../support/pages/signIn.pageObject';
-import HomePageObject from '../support/pages/home.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject'
+import HomePageObject from '../support/pages/home.pageObject'
 
-const signInPage = new SignInPageObject();
-const homePage = new HomePageObject();
+const signInPage = new SignInPageObject()
+const homePage = new HomePageObject()
 
 describe('Sign In page', () => {
-  let user;
+  let user
 
-  before(() => {
-    cy.task('db:clear');
-    cy.task('generateUser').then((generateUser) => {
-      user = generateUser;
-    });
-  });
+  beforeEach(() => {
+    cy.task('db:clear')
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser
+    })
+  })
 
   it('should provide an ability to log in with existing credentials', () => {
-    signInPage.visit();
-    cy.register(user.email, user.username, user.password);
+    signInPage.visit()
+    cy.register(user.email, user.username, user.password)
 
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
-    signInPage.clickSignInBtn();
+    signInPage.typeEmail(user.email)
+    signInPage.typePassword(user.password)
+    signInPage.clickSignInBtn()
 
-    homePage.assertHeaderContainUsername(user.username);
-  });
+    homePage.assertHeaderContainUsername(user.username)
+  })
 
   it('should not provide an ability to log in with wrong credentials', () => {
-
-  });
-});
+    signInPage.visit()
+    signInPage.typeEmail(user.email)
+    signInPage.typePassword(user.password)
```

Comment:
The test for logging in with wrong credentials uses the correct password. Modify the password to an incorrect one to properly test this scenario.